draft-ietf-quic-recovery.md

```diff
@@ -990,17 +989,20 @@ congestion without depending on PTO expiration.
 
 ### Establishing Persistent Congestion
 
-A sender establishes persistent congestion on receiving an acknowledgement if at
-least two ack-eliciting packets are declared lost, and:
+A sender establishes persistent congestion after the receipt of an
+acknowledgement if at least two ack-eliciting packets are declared lost, and:
 
-* all packets, across all packet number spaces, sent between these two send
-  times are declared lost;
+* all packets, across all packet number spaces, sent between the send times of
+  these two packets are declared lost;
```

Comment:
This is an existing problem, but I'd suggest
```suggestion
  two ack-eliciting packets are declared lost;
```

Reply:
All three bullet points referred to the "at least two ack-eliciting packets" (1) mentioned in the text before the bullet points. In my understanding of the new text, the ack-eliciting packets (2) mentioned in the first bullet point could be other packets than (1). Maybe (2) are not even declared lost.
Was this the intention or is my understanding just wrong?

Reply:
I think you're right -- I've sent out #4429